index.html

```diff
@@ -395,57 +395,17 @@ <h2 id="setup">Setup</h2>
-            <!-- Select Components -->
-            <li>Click on "Next".</li>
-            <!-- Adjusting your PATH environment -->
-            <li>
-              <strong>
-                Keep "Use Git from the Windows Command Prompt" selected and click on "Next".
-              </strong>
-                If you forgot to do this programs that you need for the workshop will not work properly.
-                If this happens rerun the installer and select the appropriate option.
-            </li>
-            <!-- Choosing the SSH executable -->
-            <li>Click on "Next".</li>
-            <!-- Configuring the line ending conversions -->
-            <li>
-              <strong>
-                Keep "Checkout Windows-style, commit Unix-style line endings" selected and click on "Next".
-              </strong>
-            </li>
-            <!-- Configuring the terminal emulator to use with Git Bash -->
-            <li>
-              <strong>
-                Keep "Use Windows' default console window" selected and click on "Next".
-              </strong>
-            </li>
-            <!-- Configuring experimental performance tweaks -->
-            <li>Click on "Install".</li>
-            <!-- Installing -->
-            <!-- Completing the Git Setup Wizard -->
-            <li>Click on "Finish".</li>
-          </ol>
-        </li>
-        <li>
-          If your "HOME" environment variable is not set (or you don't know what this is):
-          <ol>
-            <li>Open command prompt (Open Start Menu then type <code>cmd</code> and press [Enter])</li>
-            <li>
-              Type the following line into the command prompt window exactly as shown:
-              <p><code>setx HOME "%USERPROFILE%"</code></p>
-            </li>
-            <li>Press [Enter], you should see <code>SUCCESS: Specified value was saved.</code></li>
-            <li>Quit command prompt by typing <code>exit</code> then pressing [Enter]</li>
-          </ol>
-        </li>
+        <li>Download <a href="windows-installer/cygwin-installer.ps1">our script installer</a>.</li>
+	<li>Open your <code>Downloads</code> directory with File Explorer.</li>
+	<li>Right click on <code>cygwin-installer.ps1</code>.</li>
+	<li>On the menu ilustrate below, select "Properties".<img src="fig/right-click.png"></li>
+	<li>On the "Properties" window ilustrate below, check the "Unblock" box and click "OK".<img src="fig/properties.png"></li>
+	<li>Right click on <code>cygwin-installer.ps1</code>.</li>
+	<li>On the menu ilustrate below, select "Run with PowerShell".<img src="fig/right-click.png"></li>
+	<li>A blue screen will show up. <strong>Wait</strong> until the "User Account Control", ilustrate below, show up. Select "Yes".<img src="fig/user-account-control.png"></li>
+	<li><strong>Wait</strong> until a window, ilustrate below, show up with a progress bar. Wait the progress bar complete and the window close.<img src="fig/cygwin-installing.png"></li>
```

Comment:
grammar: "show up with a progress bar. Wait the progress bar complete and the window close." -> "shows up with a progress bar. Wait the progress bar completes and the window closes."